keyboards/splitkb/aurora/sweep/keymaps/flinguenheld/keymap.c

```diff
@@ -0,0 +1,127 @@
+/* Copyright 2023 Florent Linguenheld (@FLinguenheld)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+
+#include "keycodes.h"
+#include "features/auto_shift.c"
+#include "features/combo.c"
+#include "features/leader.c"
+#include "oled/logos.c"
+#include "oled/oled.c"
```

Comment:
These should be added via rules.mk, and `SRC += (file names)`.

If needed, header files should be used included (eg, `auto_shift.h`)